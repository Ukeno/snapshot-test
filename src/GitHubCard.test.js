import Profile from './GitHubCard'
import renderer from 'react-test-renderer'

test('renders a snapshot of the profile', () => {
  const tree = renderer.create(<Profile/>).toJSON()
  expect(tree).toMatchSnapshot()
})