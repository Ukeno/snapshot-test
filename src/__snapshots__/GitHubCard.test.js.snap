// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of the profile 1`] = `
<div
  className="card"
  style={
    Object {
      "backgroundColor": "lightblue",
      "height": "40.3em",
      "margin": "auto",
      "width": "20em",
    }
  }
>
  <img
    alt="Cali"
    className="card-img-top"
    src="Cali.jpg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Portfolio
    </div>
    <p
      className="card-text"
    >
      Hello my name is Kenny. At the moment i'm learning software development.
    </p>
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Button
    </button>
  </div>
</div>
`;
